zh_CN/code-of-conduct.md

```diff
@@ -0,0 +1,37 @@
+___
+**备注**
+
+这是[code-of-coduct.md](https://github.com/sindresorhus/ava/blob/master/code-of-coduct.md)的简体中文翻译。这个[链接](https://github.com/sindresorhus/ava/compare/89767ec3b6174e59d37faaadb50cfa3c0d58bda6...master#diff-d3030a18b089fdb1fbfabf6e75e4aef0)用来查看本翻译与AVA 的master 分支是否有差别（如果你没有看到`code-of-coduct.md`发生变化，那就意味着这份翻译文档是最新的）。
```

Comment:
There have spaces after English, so I think no need for add spaces before. I think this does not affect the reading.